lib/trino-filesystem-cache-alluxio/src/main/java/io/trino/filesystem/alluxio/AlluxioCoordinatorFileSystemCache.java

```diff
@@ -49,6 +52,7 @@ public AlluxioCoordinatorFileSystemCache(
             alluxioFileSystemCache = Optional.of(new AlluxioFileSystemCache(tracer, config, statistics));
         }
         else {
+            log.warn("Alluxio file caching disabled on a Coordinator");
```

Comment:
No need to log as warning message in my opinion. I would change to debug. 
```suggestion
            log.debug("File system cache is disabled on a coordinator");
```